src/bootlint.js

```diff
@@ -162,6 +164,51 @@ var LocationIndex = _location.LocationIndex;
         return runs;
     }
 
+    /**
+     * This function retrieves the browser window object and sets it to a variable.
+     * @returns {window object}
+     */
+    function getBrowserWindowObject() {
+        var theWindow = null;
+        try {
+            /*eslint-disable no-undef, block-scoped-var */
+            theWindow = window;// jshint ignore:line
+            /*eslint-enable no-undef, block-scoped-var */
+        }
+        catch (e) {
+            // deliberately do nothing
+        }
+
+        return theWindow;
+    }
+
+    function getSemverFromURI(pathSegments) {
+        return pathSegments.map(function (segment) {
+            var match = segment.match(/^\d+\.\d+\.\d+$/);
+            return match ? match[0] : null;
+        }).filter(function (match) {
+            return match !== null;
+        });
+    }
+
+    function getSemverFromPlugin(PLUGINS, globaljQuery) {
```

Comment:
PLUGINS should no longer be an argument to this function. It will just access it as a global car.